tests/ats/README.md

```diff
@@ -9,7 +9,7 @@ We want to ensure this app can be installed without any issues, so we're executi
 - Install [linkerd2-cni-app](https://github.com/giantswarm/linkerd2-cni-app) from giantswarm-test catalog
 - Install this chart
 - Install a test app ?
-- Execute `linkerd check --cni-namespace linkerd2-cni-app --linkerd-namespace linkerd2-app --output json --proxy --namespace test-app-namespace`
+- Execute `linkerd check --cni-namespace linkerd2-cni --linkerd-namespace linkerd --output json --proxy --namespace test-app-namespace`
```

Comment:
I know you only wanted to remove the `-app` suffix, but as far as I remember the CNI is running in `linkerd-cni`.

Reply:
aha! that's right. thanks!